.github/workflows/unit-test-workflow.yml

```diff
@@ -8,6 +8,7 @@ on:  # yamllint disable-line rule:truthy
       - agw-workflow
       - golang-build-test
       - nms-workflow
+      - Domain proxy workflow
```

Comment:
The other workflows are linked via file name (without extension). Does this work also with the name of the workflow? Even if, it is more consistent if the file name is used and imho more robust (renaming the workflow name does not cause failures here).

Reply:
See e.g., `golang-build-test.yml` has the name `Golang Build & Test`.

Reply:
Thank you for clarifying. The naming still confuses me - but I think this is easier to change in another PR if this is consistent for now.